lte/gateway/c/oai/tasks/amf/Registration.cpp

```diff
@@ -206,7 +202,16 @@ int amf_proc_registration_request(
     OAILOG_FUNC_RETURN(LOG_AMF_APP, rc);
   }
 
-  rc = amf_registration_run_procedure(&ue_m5gmm_context->amf_context);
+  /*
+   * Execute Initial identity Request from AMF to UE
+   */
+  // rc = state_handle_message_ue_1(
+  //    ue_m5gmm_global_context.mm_state, STATE_EVENT_REG_REQUEST, SESSION_NULL,
+  //    &ue_m5gmm_global_context, &ue_ctx.amf_context);
+  rc = state_handle_message_ue_1(
```

Comment:
what is the significance of the suffix '1'?

Reply:
we are using separate message handle functions for handlers that's why we kept as suffix 1,Now the API names got modified and removed the suffix 1 stuff.